spec/site/tlc/alarm_spec.rb

```diff
@@ -20,7 +20,66 @@
   # the test is recommended.
 
   describe 'Alarm' do
-    before { skip "Don't yet have a reliable way of triggering alarms" }
+        
+    # Validate that a detector logic fault A0302 raises and removed.
+    # 1. Given the site is connected
+    # 2. We trigger an alarm by setting S0003 input 152 to True
+    # 3. Then we should receive an active alarm
+    # 4. We remove an alarm by setting S0003 input 152 to False
+    # 5. Then we should receive an inactive alarm
+
+    specify 'A0301 is raised when S0003 input 152 is forced to True', :script, sxl: '>=1.0.7' do |example|
+      Validator::Site.connected do |task,supervisor,site|
+
+        # Alarm is raised by setting S0003 input 152 to True
+        prepare task, site
+        ensure_input('True', 152) do
```

Comment:
as mentioned before, using block with ensure_input() doesn't make sense, since the method doesn't use the block